snow/engine/snowman/bootstrap/bootstrapper_test.go

```diff
@@ -1156,12 +1161,15 @@ func TestRestartBootstrapping(t *testing.T) {
 
 	require.NoError(bs.Ancestors(context.Background(), peerID, blk4RequestID, [][]byte{blkBytes4}))
 
-	require.Equal(snow.NormalOp, config.Ctx.State.Get().State)
+	require.Equal(snow.Bootstrapping, config.Ctx.State.Get().State)
 	require.Equal(choices.Accepted, blk0.Status())
 	require.Equal(choices.Accepted, blk1.Status())
 	require.Equal(choices.Accepted, blk2.Status())
 	require.Equal(choices.Accepted, blk3.Status())
 	require.Equal(choices.Accepted, blk4.Status())
+
+	require.NoError(bs.ForceAccepted(context.Background(), []ids.ID{blkID4}))
```

Comment:
maybe a small comment to easy up UTs maintenance?
Something like that we confirm the latest downlaoded block twice to make sure bootstrap is marked as done.
It could even be made once at the top of this test file

Reply:
I added the comment to the actual bootstrapper struct